src/schema/v1/ecommerce/set_order_payment_mutation.ts

```diff
@@ -47,7 +47,8 @@ export const SetOrderPaymentMutation = mutationWithClientMutationId<
       mutation setOrderPayment($orderId: ID!, $creditCardId: String!) {
         ecommerceSetPayment(input: {
           id: $orderId,
-          creditCardId: $creditCardId,
+          paymentMethod: CREDIT_CARD,
+          paymentMethodId: $creditCardId,
```

Comment:
Updating the implementation of v1 so the tests pass with this breaking change. This is a good reminder that we should see where we are at with deprecating/retiring v1.